test/spec_helper.rb

```diff
@@ -26,7 +26,9 @@
 
 if ENV["COVERAGE"]
   require "simplecov"
-  SimpleCov.start
+  SimpleCov.start do
+    add_filter "/test/"
```

Comment:
Moreover the coverage really increased, since we have tests for the dialogs now. But they are not executed in travis.